tf/modules/iam/github_actions.tf

```diff
@@ -0,0 +1,27 @@
+locals {
+  tags = {
+    Team = "okctl-hello"
+    Env  = "dev"
+  }
+}
+
+# For thumbprint_list value: see `tf/bin/oidc.sh`
+resource "aws_iam_openid_connect_provider" "github_oidc" {
+  url             = "https://token.actions.githubusercontent.com"
+  client_id_list  = ["sts.amazonaws.com"]
+  thumbprint_list = ["6938fd4d98bab03faadb97b34396831e3780aea1"]
```

Comment:
The current expiry date of the certificate of github.com is "Thursday, March 31, 2022 at 1:59:59 AM", which probably means this github action will stop working at that date. How should we deal with this?

Reply:
`tf/bin/oidc.sh` will generate a new fingerprint, but that needs to be done manually.
In a perfect world there would be a lambda function doing a validation and looking at expiration, then notifying the user :)
But for now I'll update the documentation a bit more